doc/iterations/2024.5/shapes/guarantee_view_models_compatibility_between_versions.adoc

```diff
@@ -0,0 +1,39 @@
+= Add the support for view models migration
```

Comment:
- This should not be specific to view models but it will be specific to EMF ones
- Please update the name of the file to match its title